debug.log

```diff
@@ -0,0 +1,307 @@
+[1016/011831.005:ERROR:process_reader_win.cc(123)] NtOpenThread: {Access Denied} A process has requested access to an object, but has not been granted those access rights. (0xc0000022)
```

Comment:
hey @Seanhaoxiang - could you delete this file or gitignore it and amend?  We don't want local log files github.